app/assets/javascripts/discourse-markdown-it/addon-main.cjs

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+const { addonV1Shim } = require('@embroider/addon-shim');
+module.exports = addonV1Shim(__dirname);
```

Comment:
Strictly speaking, this isn't very necessary when the only consumer is an embroider app, but this is the general pattern